Golem.Tests/Utils.cs

```diff
@@ -152,11 +218,37 @@ public async Task InitializeAsync()
             _requestorAppKey = _requestor.getTestAppKey();
         }
 
+        public async Task StartGolem(IGolem golem, String golemPath, ChannelReader<GolemStatus> statusChannel)
```

Comment:
`StartGolem` should take the smallest context possible. Creating `statusChannel` internally would make this function more flexible.

Note:
We discussed with @pwalski that we don't know how channel will handle case with multiple isntances of the object, so it's better to leave the code as is.